cardano-node/src/Cardano/Tracing/Metrics.hs

```diff
@@ -87,3 +99,60 @@ instance All HasKESMetricsData xs => HasKESMetricsData (HardForkBlock xs) where
              => WrapForgeStateInfo blk
              -> K KESMetricsData blk
       getOne = K . getKESMetricsData (Proxy @blk) . unwrapForgeStateInfo
+
+-- | This structure stores counters of blockchain-related events,
+--   per individual forge thread.
+--   These counters are driven by traces.
+data ForgingStats
+  = ForgingStats
+  { fsTxsProcessedNum               :: !(IORef Word64)
+  , fsState                         :: !(IORef (Map ThreadId ForgeThreadStats))
+  }
+
+data ForgeThreadStats = ForgeThreadStats
+  { ftsFirstSlot                    :: !SlotNo
+  , ftsLeadershipChecksInitiated    :: !Word64
+  , ftsNodeCannotForgeNum           :: !Word64
+  , ftsNodeIsLeaderNum              :: !Word64
+  , ftsNodeNotLeaderNum             :: !Word64
+  , ftsBlocksForgedNum              :: !Word64
+  , ftsBlocksForgedInvalidNum       :: !Word64
+  , ftsSlotsMissedNum               :: !Word64
```

Comment:
These `::` are aligned across the two data types, which I'm not sure is useful. Moreover, the smallest whitespace gap before a `::` is more than 1 space. In other words, I would expected `ftsLeadershipChecksInitiated :: !Word64` with 1 space before the `::`.

Reply:
The idea behind the excess whitespace is to avoid re-indentation when something longer is added.  In the case of `ForgingStats`, though, even that excess was excessive, yes : -) 

But yes, removing all  that.